tsconfig.json

```diff
@@ -28,5 +28,5 @@
     "skipLibCheck": true,
     "forceConsistentCasingInFileNames": true
   },
-  "exclude": ["node_modules", "dist", "__tests__"]
+  "exclude": ["node_modules", "dist"]
```

Comment:
You can include `callbacks.ts` on `setupFilesAfterEnv` in order to run before all the tests and in this way avoid import this file in each file, Idk if that is the expected behavior, isn't it?

Reply:
It is!! I just tried it and it looks like it worked like a charm! Nice catch!

Reply:
Added `setupFilesAfterEnv` into `jest.config.js` 👌